rwiki/rwiki-model/src/java/uk/ac/cam/caret/sakai/rwiki/model/RWiki.hbm.xml

```diff
@@ -103,5 +103,6 @@
 		<property name="publicWrite" type="boolean" />
 		<property name="sha1" length="64" />
 		<property name="revision"  type="int"/>
+		<property name="groups" type="string" length="4000"/>
```

Comment:
Just a comment: not sure if dealing with groups as a string is the best way to manage all this, but if we follow this approach we have to be careful with all the potential cases where the string might be troublesome (performance, group deletion...)

(Plus this will need a sakai-reference PR for conversion)

Reply:
Yes, I am right with you, but i don't know what is the best way to deal with the groups. So, if you know another way, I accept suggestions
Thanks for all the help

Reply:
I think it's fine like this considering the tool is not very active code-wise anymore. Already closed the conversation

Reply:
Why not materialized_clob?

<property name="groups" type="materialized_clob" column="INSTRUCTION" not-null="false" />

Also GROUPS is a reserved group in MySQL 8
https://dev.mysql.com/doc/mysqld-version-reference/en/keywords-8-0.html

I think we can name it pageGroups

<property name="pageGroups" type="materialized_clob" not-null="false" />

Reply:
What do you store here? GroupIds separated by comma?

Reply:
Yes